ntbs-service/Models/Validations/ValidationHelper.cs

```diff
@@ -44,6 +44,8 @@ public static class ValidationMessages
         #region Patient Details
         public const string NhsNumberLength = "{0} needs to be 10 digits long";
         public const string InvalidNhsNumber = "This {0} is not valid. Confirm you have entered it correctly";
+        // We think users trying to enter a monodigit nhs number are doing so because they do not know the nhs number.
+        public const string NhsNumberMonodigit = "This {0} is not valid. If you do not know the NHS number please select \"Not Known\"";
```

Comment:
It's weird that we're passing in arguments to these strings, as I expect that the argument would only ever be "NHS Number". I think we should either double-check and remove the argument from both, or we should include the second "NHS Number" as "{0}" for consistency.

Reply:
I'm guessing it was done as other validation error messages are used for multiple inputs. These are only used for the nhs number as expected so I've removed the arguments